autospec/git.py

```diff
@@ -117,6 +119,7 @@ def commit_to_git(path):
         "!*.tar.*.*",
         "*.zip",
         "commitmsg",
+        "list",
```

Comment:
The filename `list` is not very descriptive... Can it have a different name that better reflects its purpose?

Reply:
Actually because of how I am handling the list file generation I can likely rework even downloading the file. Thanks for pointing this out. Fixing.